libtelemetry/src/main/java/com/mapbox/android/telemetry/TelemetryUtils.java

```diff
@@ -121,20 +119,18 @@ public static String obtainUniversalUniqueIdentifier() {
   }
 
   public static String obtainApplicationState(Context context) {
-    ActivityManager activityManager = (ActivityManager) context.getSystemService(Context.ACTIVITY_SERVICE);
-    List<ActivityManager.RunningAppProcessInfo> appProcesses = activityManager.getRunningAppProcesses();
-    if (appProcesses == null) {
-      return NO_STATE;
+    String state = NO_STATE;
+    switch (AppStateUtils.getAppStateFromActivityManager(context)) {
```

Comment:
I may not see full picture but couldn't we actually expose and use `AppStateUtils.getAppStateFromActivityManager(context)` instead of `obtainApplicationState`? This mapping here seems to be actually almost 1 to 1 (at least looking at the diff here).

Reply:
Historical reasons. :)  We can expose `AppStateUtils.getAppStateFromActivityManager(context)` but I would avoid removing `obtainApplicationState` cause it's a breaking change.

Reply:
Couldn't we do something like
```
public static String obtainApplicationState(Context context) {
  return appStateUtils.getAppStateFromActivityManager(context);
}
```
if I get the mapping correctly?

Reply:
Well, NO_STATE is an empty string, but others are one-to-one, simplified.